### ISSUE TITLE
No autoload on change when using WSL2 (Ubuntu).

### ISSUE THREAD

**Issue by lhao03** [getzola/zola#1440]
# Bug Report

## Environment
Windows, but using WSL2 (Ubuntu)
Zola version: 0.13

## Expected Behavior
Every time I save a file the site should reload and show the new changes. When I was on WSL1 (Ubuntu), Zola was reloading for every time I changed something and saved.

## Current Behavior
I have to stop the current ```zola serve``` and  rerun ```zola serve``` every time I make a change to see the change reflected on my website.

## Step to reproduce
Open a Zola project and then run ```zola serve``` in WSL2. Change something and save the file. The site does not reload. Maybe it's because on the Linux file system my website is stored at "/mnt/c/Users/[username]/codes/personal-site-v3" but it's stored on my Windows file system at "C:\Users\[username]\Codes\personal-site-v3". However, the reload was working when I was on WSL1.

**Comment by Keats** [comment]
Sorry I don't have a machine with WSL2. Can someone who has help debug that?
All the live reload code lives in https://github.com/getzola/zola/blob/master/src/cmd/serve.rs

**Comment by lhao03** [comment]
I figured out what was wrong! I'm not sure how this was working in WSL1, but the WSL and Windows have different file systems (https://docs.microsoft.com/en-us/windows/wsl/compare-versions). Once I moved my website to  the WSL file system ("/home/.../personal-site-v3") the auto reload is working again! For WSL2 the website has to be stored in the Linux file system and not the Windows file system for the auto reload to work.
